tfprotov6/state_store.go

```diff
@@ -57,7 +57,7 @@ type ReadStateBytesStream struct {
 }
 
 type WriteStateBytesStream struct {
-	Chunks iter.Seq[WriteStateByteChunk]
+	Chunks iter.Seq2[WriteStateByteChunk, error]
```

Comment:
TODO: Refactor to use struct that contains chunk and error as fields, versus using Seq2